dev/deps/spark-deps-hadoop-2.2

```diff
@@ -122,7 +122,7 @@ jsr305-1.3.9.jar
 jta-1.1.jar
 jtransforms-2.4.0.jar
 jul-to-slf4j-1.7.16.jar
-kryo-2.21.jar
+kryo-shaded-3.0.3.jar
```

Comment:
Thanks to this shading, we no longer need to add exclusions for Kryo's ASM dependencies.